src/main/scala/org/broadinstitute/dsde/firecloud/MethodsClient.scala

```diff
@@ -43,24 +44,30 @@ class MethodsClient(requestContext: RequestContext) extends Actor {
       addHeader(Cookie(requestContext.request.cookies)) ~> sendReceive
 
     val responseFuture: Future[HttpResponse] = pipeline { Get(FireCloudConfig.Methods.methodsListUrl) }
+    val corsHeaders = List(
+      RawHeader("Access-Control-Allow-Credentials", "true"),
+      RawHeader("Access-Control-Allow-Origin", "http://local.broadinstitute.org:8000")
```

Comment:
how about allowing * instead of local.b.o? That would allow https://firecloud-ci.broadinstitute.org/ to work as well. Since we require OpenAM authentication, we're protected.

Additionally, looks like we need to add a Vary: Origin header - see https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS#Access-Control-Allow-Origin